ocaml/typing/env.ml

```diff
@@ -2652,28 +2652,26 @@ let persistent_structures_of_dir dir =
   |> String.Set.of_seq
 
 (* Save a signature to a file *)
-let save_signature_with_transform cmi_transform ~alerts sg modname filename =
+let save_signature_with_transform cmi_transform ~alerts sg modname cu filename =
```

Comment:
`cu` seems an odd name for a `Cmi_format.kind` value

Reply:
Renamed to `kind`